Formula/lib/libtirpc.rb

```diff
@@ -9,10 +9,23 @@ class Libtirpc < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux: "91acf8d2991b6c027d780f8662e81f154c6101d651fb80bf16db5e6d8ba0696f"
   end
 
-  depends_on "krb5"
-  depends_on :linux
+  # the build depends are needed while we need to regenerate configure
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+
+  uses_from_macos "krb5"
+  patch :DATA # all patches were sent to libtirpc-devel during the 1.3.6 cycle
 
   def install
+    # the autoreconf and environment overrides can go away when the patches
+    # that are inlined here are accepted or otherwise implemented upstream
+    ENV["ACLOCAL"] = "aclocal"
+    ENV["AUTOMAKE"] = "automake"
```

Comment:
Why do we need to specify these environment variables?

Reply:
because otherwise something is deciding am__api_version is 1.16 and trying to run aclocal-1.16; but current aclocal is 1.17, so there is no aclocal-1.16. the environment only matters during build and you don't need 1.16; 1.17 (or newer) will work fine.

Reply:
you also won't need these once upstream takes whatever changes. they're only needed because the configure script bundled in the release needs to be rebuilt from patches

Reply:
also of note, other ports do this. see f/flac123.rb

Reply:
changed it so this is hopefully not required